addons/website_event/controllers/main.py

```diff
@@ -153,11 +153,16 @@ def dom_without(without):
 
```

Comment:
can you be a little more specific in your commit message ?
like talking about the 3 first attendees, adding organizer and location, etc..